indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -161,8 +161,8 @@
 }
 
 .menu-entries {
-    width: 370px;
-    padding: 5px 10px 5px 5px;
+    max-width: 370px;
```

Comment:
I don't see any problem when removing this rule completely. Same with the one in line `325`. The more we can avoid hard-coded pixel values, the better.